packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -164,8 +164,8 @@ async function transformTransaction(tx: Transfer): Promise<MergedTransaction> {
     return transformDeposit(
       await updateAdditionalDepositData({
         depositTx: tx,
-        l1Provider: parentChainProvider,
-        l2Provider: childChainProvider
+        parentProvider: parentChainProvider,
```

Comment:
We can initialize `parentChainProvider` as `parentProvider` (similarly with child)  to simplify this like done in teleport/helper.ts